single-line-diagram/single-line-diagram-core/src/test/resources/TestSldClassSubstationMetadata.json

```diff
@@ -536,58 +536,36 @@
   "electricalNodeInfos" : [ ],
   "busInfos" : [ ],
   "layoutParams" : {
-    "voltageLevelPadding" : {
-      "left" : 20.0,
-      "top" : 60.0,
-      "right" : 20.0,
-      "bottom" : 60.0
-    },
-    "diagramPadding" : {
-      "left" : 20.0,
-      "top" : 20.0,
-      "right" : 20.0,
-      "bottom" : 20.0
-    },
     "verticalSpaceBus" : 25.0,
     "horizontalBusPadding" : 20.0,
     "cellWidth" : 50.0,
     "externCellHeight" : 250.0,
     "internCellHeight" : 40.0,
     "stackHeight" : 30.0,
-    "showGrid" : false,
-    "tooltipEnabled" : false,
-    "showInternalNodes" : false,
-    "scaleFactor" : 1.0,
-    "drawStraightWires" : false,
     "horizontalSnakeLinePadding" : 20.0,
     "verticalSnakeLinePadding" : 25.0,
-    "feederInfosOuterMargin" : 20.0,
     "spaceForFeederInfos" : 50.0,
-    "diagramName" : null,
-    "avoidSVGComponentsDuplication" : false,
     "adaptCellHeightToContent" : true,
     "maxComponentHeight" : 12.0,
     "minSpaceBetweenComponents" : 15.0,
     "minExternCellHeight" : 80.0,
-    "labelCentered" : false,
-    "labelDiagonal" : false,
-    "angleLabelShift" : 15.0,
-    "addNodesInfos" : false,
-    "feederInfoSymmetry" : false,
-    "cssLocation" : "INSERTED_IN_SVG",
-    "svgWidthAndHeightAdded" : false,
-    "useName" : false,
-    "feederInfosIntraMargin" : 10.0,
-    "busInfoMargin" : 0.0,
+    "voltageLevelPadding" : {
+      "left" : 20.0,
+      "top" : 60.0,
+      "right" : 20.0,
+      "bottom" : 60.0
+    },
+    "diagramPadding" : {
+      "left" : 20.0,
+      "top" : 20.0,
+      "right" : 20.0,
+      "bottom" : 20.0
+    },
     "busbarsAlignment" : "FIRST",
     "componentsOnBusbars" : [ "DISCONNECTOR" ],
-    "languageTag" : "en",
-    "voltageValuePrecision" : 1,
-    "powerValuePrecision" : 0,
-    "angleValuePrecision" : 1,
-    "currentValuePrecision" : 0,
-    "displayCurrentFeederInfo" : false,
-    "undefinedValueSymbol" : "—",
-    "removeFictitiousSwitchNodes" : false
```

Comment:
You've split the parameters in two parts, but you didn't add the `SvgParameters` in `GraphMetadata`. Are you sure they're of no use? I think we'd better add them in the metadata, so we keep the same information as before.